course_discovery/apps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -135,6 +141,24 @@ def test_list(self):
             self.serialize_course_run(CourseRun.objects.all().order_by(Lower('key')), many=True)
         )
 
+    def test_list_edx_org_short_name_filter(self):
+        """
+        Verify the endpoint returns a filtered list of course runs based on edx organization.
+        """
+        course_run_api_url_with_org_filter = '{course_run_api_url}?org={edx_org_short_name}'.format(
+            course_run_api_url=reverse('api:v1:course_run-list'),
+            edx_org_short_name=self.edx_org_short_name
+        )
+        expected = self.serialize_course_run(
+            CourseRun.objects.filter(course_authoring_organizations__key=self.edx_org_short_name).order_by(Lower('key')),
+            many=True
+        )
+
+        response = self.client.get(course_run_api_url_with_org_filter)
+        actual = response.data['results'],
```

Comment:
`course_runs_from_response`
Also we need to verify status code first then fetch results. If there is 500 how can we get results?

Reply:
Done